packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -50,6 +51,19 @@ async function setupBrowser() {
 	await setBrowserViewport( 'large' );
 }
 
+/**
+ * Verifies that the expected theme for the tests is active.
+ */
+async function verifyActiveTheme() {
+	await page.goto(
+		createURL( '' )
+	);
```

Comment:
What does this ensure, and can we achieve the same perhaps with a more semantic use of Puppeteer's API?

Reply:
It takes you to the home page in the front end of the site so that the theme's stylesheets will be loaded.

>a more semantic use of Puppeteer's API?

What were you thinking?

Reply:
I was thrown off by my own expectation of not having to navigate anywhere. Indeed, with a one-off requirement specially made for Twenty Twenty it would be enough to look for `#twentytwenty-block-editor-styles-css` in the editor itself.

What you're doing is more generic, since the existence of `#foo-style-css` in the front end is guaranteed by WordPress itself, not the theme implementation. There's no such guarantee for the back end, alas, as editor styles are optional.

In short: disregard my comment about anything more semantic.